src/services/controller.service.ts

```diff
@@ -16,6 +17,12 @@ export class ControllerService {
       return;
     }
 
+    try {
+      await this._storageService.connectToDB();
```

Comment:
Do we need to call this again?

It's already called in bot.ts, which precedes this

Reply:
i changed `connectToDB` to return a cached promise. that way we only do the work once but we can also enforce that it's been done